perfmetrics/scripts/testing_on_gke/examples/utils/utils_test.py

```diff
@@ -118,17 +98,20 @@ def test_resource_limit(self):
         {'nodeType': 'n2-standard-1', 'expected_error': True},
         {'nodeType': 'unknown-machine-type', 'expected_error': True},
     ]
+
     for input in inputs:
-      self.assertEqual(dict, type(input))
-      try:
+
+      if input['expected_error']:
+
+        with self.assertRaises(utils.UnknownMachineTypeError):
+          resource_limits = utils.resource_limits(input['nodeType'])
```

Comment:
What's the point of this assignment?
```suggestion
          utils.resource_limits(input['nodeType'])
```

Reply:
it seems like an unwanted assignment left over from the split into `if-else` in this change.

Reply:
removed